### PR TITLE
Feature/t3d

### PR DESCRIPTION
**Pull request by nickgal**

### CODE REVIEW

File: LanternExtractor/EQ/Archive/ArchiveType.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace LanternExtractor.EQ.Archive
+{
+    public enum ArchiveType
+    {
+        Unknown,
```
Comment (danwilkins) [inline LanternExtractor/EQ/Archive/ArchiveType.cs]: How often do we actually run into an `Unknown` type archive? Also, can you assign numerical values to these.

File: LanternExtractor/EQ/Wld/DataTypes/RenderInfo.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        public static RenderInfo Parse(BinaryReader reader, List<WldFragment> fragments)
+        {
+            var renderInfo = new RenderInfo
+            {
+                Flags = reader.ReadInt32()
+            };
+
+            var ba = new BitAnalyzer(renderInfo.Flags);
+
+            var hasPen = ba.IsBitSet(0);
```
Comment (danwilkins) [inline LanternExtractor/EQ/Wld/DataTypes/RenderInfo.cs:28]: Do we know yet what this does? See it in the later WLD files, too.

File: LanternExtractor/EQ/Wld/Helpers/CharacterFixer.cs [hunk 3]
```diff
@@ -389,6 +390,29 @@ private void FixHalasFemale()
...
         }
 
+        /// <summary>
+        /// Early beta beetles have SPI named bones
+        /// </summary>
+        private void FixBetaBeetle()
+        {
+            var skeleton = _wld.GetFragmentByName<SkeletonHierarchy>("BET_HS_DEF");
+
+            if (skeleton == null)
```
Comment (danwilkins) [inline LanternExtractor/EQ/Wld/Helpers/CharacterFixer.cs:400]: This is a good pattern we can apply to the others fixes which iterate through and continue on mismatched names.

**Review by danwilkins — APPROVED** [review]

**Comment by danwilkins** [comment]
Excellent work on this!

File: LanternExtractor/EQ/Wld/DataTypes/RenderInfo.cs (hunk 2 above)
Comment (nickgal) [reply LanternExtractor/EQ/Wld/DataTypes/RenderInfo.cs:28]: There are a couple functions suggesting something related to palette & transparency but I haven't figured out how it all comes together yet. `t3dSetRenderContextTransparentPen` `t3dPaletteClosestPenToColor`

File: LanternExtractor/EQ/Archive/ArchiveType.cs (hunk 1 above)
Comment (nickgal) [reply LanternExtractor/EQ/Archive/ArchiveType.cs]: Realistically I think unknown will really only get hit when non archive files get renamed to one of the archive extensions or when the files are damaged to the point where the the first 4 (or second 4) bytes are incorrect.

**Pull request merged**